tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -5,7 +5,7 @@ const commonElements = require('../common/common.wdio.page');
 const sentinelUtils = require('../../../utils/sentinel');
 const utils = require('../../../utils');
 const modalPage = require('../../../page-objects/default/common/modal.wdio.page');
-const searchBox = () => $('.mm-search-bar-container input#freetext');
+const searchBox = () => $('input#freetext');
```

Comment:
Can you please review [the comment from @latin-panda](https://github.com/medic/cht-core/pull/8054/files#r1107967140) in the previous review?

Reply:
When using `const searchBox = () => $('.mm-search-bar-container input#freetext');` I'm getting:
``` 
Can't call setValue on element with selector ".mm-search-bar-container input#freetext" because element wasn't found
```